### ISSUE TITLE
Entry OCGs missing

### ISSUE THREAD

**Issue by drexvictor** [hhrutter/pdfcpu#23]
I had tried to extract the images from [this file](http://www.ipindia.nic.in/writereaddata/Portal/IPOJournal/1_2612_1/CLASS_1_-_6.pdf)
and get an error. The following is the trace from debug.

DEBUG: 2018/05/16 21:22:18 validateIndRefArrayEntry begin: entry=OCGs
DEBUG: 2018/05/16 21:22:18 validateArrayEntry begin: entry=OCGs
Fatal: dict=optContentPropertiesDict required entry=OCGs missing
github.com/hhrutter/pdfcpu.(*PDFDict).Entry
	/src/go/src/github.com/hhrutter/pdfcpu/dict.go:87
github.com/hhrutter/pdfcpu.validateArrayEntry
	/src/go/src/github.com/hhrutter/pdfcpu/validateObjects.go:54
github.com/hhrutter/pdfcpu.validateIndRefArrayEntry
	/src/go/src/github.com/hhrutter/pdfcpu/validateObjects.go:452
github.com/hhrutter/pdfcpu.validateOCProperties
	/src/go/src/github.com/hhrutter/pdfcpu/validateOptionalContent.go:396
github.com/hhrutter/pdfcpu.validateRootObject
	/src/go/src/github.com/hhrutter/pdfcpu/validateXReftable.go:888
github.com/hhrutter/pdfcpu.validateXRefTable
	/src/go/src/github.com/hhrutter/pdfcpu/validateXReftable.go:15
github.com/hhrutter/pdfcpu.readAndValidate
	/src/go/src/github.com/hhrutter/pdfcpu/api.go:184
github.com/hhrutter/pdfcpu.readValidateAndOptimize
	/src/go/src/github.com/hhrutter/pdfcpu/api.go:195
github.com/hhrutter/pdfcpu.ExtractImages
	/src/go/src/github.com/hhrutter/pdfcpu/api.go:626
github.com/hhrutter/pdfcpu.Process
	/src/go/src/github.com/hhrutter/pdfcpu/process.go:41
main.process
	/src/go/src/github.com/hhrutter/pdfcpu/cmd/pdfcpu/main.go:257
main.main
	/src/go/src/github.com/hhrutter/pdfcpu/cmd/pdfcpu/main.go:113
runtime.main
	/usr/lib/go-1.10/src/runtime/proc.go:198
runtime.goexit
	/usr/lib/go-1.10/src/runtime/asm_amd64.s:2361

**Comment by hhrutter** [comment]
Looks like this PDF does not follow the spec,
Who is the creator of this file?
Do you think you can share a sample file for analysis?

**Comment by drexvictor** [comment]
The file is at http://www.ipindia.nic.in/writereaddata/Portal/IPOJournal/1_2612_1/CLASS_1_-_6.pdf

**Comment by hhrutter** [comment]
Fixed!
I think we can add this missing info as relaxed valid.
I am in the middle of cutting a new release due by this weekend
but if you can't wait you can patch the sources like so:

Replace lines 395-399 in validateOptionalContent.go with the following:
```
// "OCGs" required array of already written indRefs
r := true
if xRefTable.ValidationMode == ValidationRelaxed {
	r = false
}
_, err = validateIndRefArrayEntry(xRefTable, dict, dictName, "OCGs", r, sinceVersion, nil)
if err != nil {
	return err
}
```
then you do 
```
cd $GOPATH/src/github.com/hhrutter/pdfcpu/cmd/pdfcpu
go install
```

**Comment by hhrutter** [comment]
fixed by version 0.1.12
